sys-auth/yubikey-personalization-gui/yubikey-personalization-gui-3.1.25.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit flag-o-matic qmake-utils
```

Comment:
missing desktop.eclass (you are using `doicon` and `domenu` in `src_install`

Reply:
That's from your diff... ;) 
I'll see to these changes.